### ISSUE TITLE
vocal part: semi supervised training loss come to negative

### ISSUE THREAD

**Issue by li2dong4** [Music-and-Culture-Technology-Lab/omnizart#45]
Hi, thanks for your great job.
The loss became negative after the first epoch while I trained a new model with labeled and unlabeled data. Is this right？

**Comment by BreezeWhite** [comment]
That's kind of weird. But as I remember, the training is pretty unstable for the vocal module. You could stop the training and start again until you see the normal training loss.
